int32 frameid
cnn_box[] cnn_box